index.js

```diff
@@ -40,7 +40,51 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("#header")
+
+header.addEventListener("click", event => {
+  toggleColor(header)
+})
+
 
 /***** Deliverable 2 *****/
+const playerForm = document.querySelector("#new-player-form")
+
+playerForm.addEventListener("submit", function(e) {
+  e.preventDefault()
+
+  const form = e.target
+    playerObj = {
+      number: form.number.value,
+      name: form.name.value,
+      nickname: form.nickname.value,
+      photo: form.photo.value,
+      likes: 0
+    }
+
+  form.reset()
+  renderPlayer(playerObj)
```

Comment:
Looks great! Just be careful on line 57, you're declaring the `playerObj` without `let` or `const` so it will become a global variable.